lib/engine.py

```diff
@@ -706,7 +706,8 @@ def filename_from_url(url):
 
 
 def gen_from_source(source):
-    """Returns a generator from a source tuple.
+    """
+    Returns a generator from a source tuple.
```

Comment:
Okay, will revert.

On Aug 9, 2016 7:20 PM, "Ethan White" <notifications@github.com> wrote:

> In lib/engine.py
> <https://github.com/weecology/retriever/pull/599#discussion_r74060880>:
>
> > @@ -706,7 +706,8 @@ def filename_from_url(url):
> >
> >
> >  def gen_from_source(source):
> > -    """Returns a generator from a source tuple.
> > +    """
> > +    Returns a generator from a source tuple.
>
> PEP257 says it should be as it was originally: https://www.python.org/dev/
> peps/pep-0257/#multi-line-docstrings
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/weecology/retriever/pull/599/files/5f3d780dac2436dc8aa47e56ef1d534d2469389e#r74060880>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGplhIdDfuT1N4ltEpX8EScHz0QXaDdxks5qeIWXgaJpZM4Jekew>
> .
>